### PR TITLE
housekeeping: cleaning code

### PR DESCRIPTION
**Pull request by fbac**
- Enforce go style in some parts.
- Add extra checks for tests.

### CODE REVIEW

File: pkg/timelock/operations_test.go [hunk 1]
```diff
@@ -138,6 +153,11 @@ func Test_isDone(t *testing.T) {
 
 func Test_isPending(t *testing.T) {
 	testWorker, _ := NewTimelockWorker(testNodeURL, testTimelockAddress, testCallProxyAddress, testPrivateKey, testFromBlock, int64(testPollPeriod), testLogger)
+
```
Comment (MStreet3) [inline pkg/timelock/operations_test.go]: maybe create a test helper for this?

```go
func newTestTimelockWorker(t, ...) *timelockWorker {
    tw, err := NewTimelockWorker(...)
    assert.NoError(t, err)
    assert.NotNil(t, tw, "testWorker is nil, are the env variables in const_test.go set?")
}
```

**Review by MStreet3 — COMMENTED** [review]
would suggest creating a test constructor that wraps the real constructor for use in tests

**Review by MStreet3 — COMMENTED** [review]
You should be able to remove the nil checks now and put them into the new constructor. The assert will make the test fail if it is nil.

**Comment by fbac** [comment]
> You should be able to remove the nil checks now and put them into the new constructor. The assert will make the test fail if it is nil.

Done, forgot completely about removing those.

File: pkg/timelock/scheduler_test.go [hunk 2]
```diff
@@ -45,7 +45,11 @@ func Test_newScheduler(t *testing.T) {
 }
 
 func TestWorker_updateSchedulerDelay(t *testing.T) {
-	testWorker, _ := NewTimelockWorker(testNodeURL, testTimelockAddress, testCallProxyAddress, testPrivateKey, testFromBlock, int64(testPollPeriod), testLogger)
+	testWorker := newTestTimelockWorker(t, testNodeURL, testTimelockAddress, testCallProxyAddress, testPrivateKey, testFromBlock, int64(testPollPeriod), testLogger)
+
+	if testWorker == nil {
```
Comment (MStreet3) [inline pkg/timelock/scheduler_test.go]: Remove too

**Review by MStreet3 — COMMENTED** [review]
One last statement to remove

File: pkg/timelock/operations_test.go [hunk 3]
```diff
@@ -96,6 +98,7 @@ func Test_isReady(t *testing.T) {
 
 func Test_isDone(t *testing.T) {
 	testWorker, _ := NewTimelockWorker(testNodeURL, testTimelockAddress, testCallProxyAddress, testPrivateKey, testFromBlock, int64(testPollPeriod), testLogger)
+
```
Comment (MStreet3) [inline pkg/timelock/operations_test.go:101]: Use the new test constructor here too

File: pkg/timelock/operations_test.go [hunk 4]
```diff
@@ -138,6 +141,7 @@ func Test_isDone(t *testing.T) {
 
 func Test_isPending(t *testing.T) {
 	testWorker, _ := NewTimelockWorker(testNodeURL, testTimelockAddress, testCallProxyAddress, testPrivateKey, testFromBlock, int64(testPollPeriod), testLogger)
+
```
Comment (MStreet3) [inline pkg/timelock/operations_test.go:144]: Use the new test constructor here too

**Review by MStreet3 — COMMENTED** [review]
Missed two tests

**Review by MStreet3 — APPROVED** [review]

**Pull request merged**